deploy/roles/eod7/tasks/install_eod.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Copy over the EOD bin file
+  copy:
+    src: "files/{{ eod_installer }}"
+#    src: "{{ eod_installer_directory }}/{{ eod_installer }}"
```

Comment:
@lel99999 This needs to be reverted since the binary cannot be placed in `files` in this repo

Reply:
Reverted.